infrastructures/infrastructure-common/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/ConnectorIaasController.java

```diff
@@ -130,7 +130,8 @@ public String createAzureInfrastructure(String infrastructureId, String clientId
                                                                                 resourceGroup,
                                                                                 region,
                                                                                 privateNetworkCIDR,
-                                                                                staticPublicIP);
+                                                                                staticPublicIP,
+                                                                                null);
```

Comment:
What is the new parameter and why is it null? Could we have another method so that we do not have to handle null objects?